update_bash_env.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+"""
+Generate a file for use by BASH_ENV
+"""
+
+import json
+from pathlib import Path
+from logging import getLogger, basicConfig
+from argparse import ArgumentParser
+from yaml import safe_load, YAMLError
+from jinja2 import Environment, FileSystemLoader
+
+formatting = json.dumps(
+    {
+        "timestamp": "%(asctime)s",
+        "namespace": "%(name)s",
+        "loglevel": "%(levelname)s",
+        "message": "%(message)s",
+    }
+)
+basicConfig(level="WARNING", format=formatting)
+LOG = getLogger("easy_infra")
+
+
```

Comment:
i'm not familiar with passing *, what are you doing here?  is there a more clear way? one with hinting?

Reply:
Requires that arguments are passed via kwargs